examples/eks/eks.tf

```diff
@@ -0,0 +1,24 @@
+// Region is AWS region, the region should support EFS
+variable "region" {
+  type = "string"
+  default = "us-west-2"
+}
+
+// Script creates a separate VPC with demo deployment
+variable "vpc_cidr" {
+  type = "string"
+  default = "172.31.0.0/16"
+}
+
+variable "cluster-name" {
+  default = "terraform-eks-demo"
+  type    = "string"
+}
+
+provider "aws" {
+  version    = "~> 2.1.0"
```

Comment:
I would recommend using 2.7.0 as this is compatible with Terraform 0.12.0. There is a fair amount of improvements in TF 0.12.0 https://www.terraform.io/upgrade-guides/0-12.html